### ISSUE TITLE
Manual Installation check outdated

### ISSUE THREAD

**Issue by vrglab** [Ayugradow/ModInstaller#130]
When you search Manually for the Installation path the Installer checks for a Folder named "Hollow_Knight_Data" where in Actuality the Folder it's searching for is now named "Hollow Knight_Data" causing the Installer to give us a error and fail.

**Comment by Tulip-0333** [comment]
No, it's called "hollow_knight_Data". It works completely fine with even with different installation directory.

**Comment by vrglab** [comment]
Actually that's true but the issue still stands.
 i figured out that this problem only Happens if you bought the game from the Microsoft store and not steam, meaning unless you buy the game from steam you can't use this Mod installer.
